app/views/sponsors/show.html.erb

```diff
@@ -1,4 +1,4 @@
-
+<% provide(:title_tag, "#{@sponsor.program_title} Overview") %>
```

Comment:
Should these be `<%=`? Same question for all the app/views/submissions/*.erb

Reply:
Works either way. No rendering here so this works. 

Should be consistent, though. Thanks for catching!